frappe/www/printview.py

```diff
@@ -97,6 +97,9 @@ def get_context(context) -> PrintContext:
 		"doctype": frappe.form_dict.doctype,
 		"name": frappe.form_dict.name,
 		"key": frappe.form_dict.get("key"),
+		"print_format": print_format.name,
```

Comment:
```suggestion
		"print_format": getattr(print_format, "name", None),
```

Or perhaps a separate check for this beforehand, since this can be NoneType